### DISCUSSION TITLE
Histoire UI

### DISCUSSION THREAD

**Discussion by victorlmneves** [histoire-dev/histoire#202] [Q&A]
Hi
Do we have a way of controlling the length of the string displayed on source code panel?
I'm having a problem with a prop that is a encoded svg that is creating a huge horizontal scroll-bar
![image](https://user-images.githubusercontent.com/17341789/180791512-578c259f-0e7b-451d-9d42-876ef01d9770.png)
![image](https://user-images.githubusercontent.com/17341789/180791655-8f0f4599-7538-4981-b64c-53c78aea10ed.png)
![image](https://user-images.githubusercontent.com/17341789/180791795-351ab29b-84fd-4b66-8fc3-170f605e8a13.png)

**Comment by Akryum** [comment]
Maybe we should force line wrapping on long lines?

**Reply by victorlmneves** [reply]
That could the faster solution, but I also created an issue suggesting some enhancements so we can have more control over the app UI
